packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -223,12 +223,36 @@ export class SchemaFactory<
 	 * If each library exporting schema picks its own globally unique scope for its SchemaFactory,
 	 * then all schema an application might depend on, directly or transitively,
 	 * will end up with a unique fully qualified name which is required to refer to it in persisted data and errors.
```

Comment:
I can't parse that block. What does this refer to? "require to refer" - what is the API / workflow that it refers to?

Reply:
I have rewritten this to be simpler so it's less of a toss up if you happen to parse the sentence in a way that works.